zerver/models.py

```diff
@@ -2250,9 +2250,9 @@ class Meta:
         abstract = True
 
 class ScheduledEmail(AbstractScheduledJob):
-    # Exactly one of user or address should be set. These are used to
+    # Exactly one of users or address should be set. These are used to
     # filter the set of ScheduledEmails.
-    user = models.ForeignKey(UserProfile, null=True, on_delete=CASCADE)  # type: Optional[UserProfile]
+    users = models.ManyToManyField(UserProfile)  # type: Manager
```

Comment:
I added a more extensive comment here explaining the fact that the values here are duplicated with `data`, since that confused me for a moment while reading this.